utils/build_system/create_test_release_jobs.py

```diff
@@ -144,18 +152,37 @@ def _log_jenkins_exception(exc):
         else:
             LOGGER.error(exc)
 
+    def load_defines(self, root: Path) -> dict[str, str]:
+        try:
+            content = (root / "_jobDefines.yaml").open()
+        except FileNotFoundError:
+            return {}
+        defines = yaml.safe_load(content)
+        return defines
+
+    @staticmethod
+    def locate_job_overrides(job_name: str, overrides: dict[str, str]) -> dict[str, str]:
+        if full_match := overrides.get(job_name):
+            return full_match
+
+        for key in overrides:
+            if re.search(re.compile(key, re.IGNORECASE), job_name):
+                return overrides.get(key, {})
+
+        return {}
+
     def create_job_tree(self, local_path: str | Path,  # pylint: disable=too-many-arguments
                         create_freestyle_jobs: bool = True,
                         create_pipelines_jobs: bool = True,
                         template_context: dict | None = None,
                         job_name_suffix: str = '-test'):
         for root, _, job_files in os.walk(local_path):
             jenkins_path = Path(root).relative_to(local_path)
+            defines = self.load_defines(Path(root))
+            job_overrides = defines.pop("overrides", {})
 
             # get display names, if available
-            display_name = jenkins_path.name
-            if '_display_name' in job_files:
```

Comment:
we should leave this as fallback, until all of those are replace in the tree..

Reply:
Okay

Reply:
Reverted